iDiff/README.md

```diff
@@ -0,0 +1,245 @@
+# iDiff
+
+[![Build
+Status](https://travis-ci.org/GoogleCloudPlatform/runtimes-common.svg?branch=master)](https://travis-ci.org/GoogleCloudPlatform/runtimes-common)
+[![codecov](https://codecov.io/gh/GoogleCloudPlatform/runtimes-common/branch/master/graph/badge.svg)](https://codecov.io/gh/GoogleCloudPlatform/runtimes-common)
```

Comment:
maybe just remove this, since we're not running this on the runtimes-common repo?